packages/ra-input-rich-text/src/index.spec.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import debounce from 'lodash/debounce';
+import {render, fireEvent, waitForElement, cleanup, getByTestId} from 'react-testing-library'
+
+import { RichTextInput } from './index';
+
+let container;
+
+jest.mock('lodash/debounce');
+describe('RichTextInput', () => {
+    beforeEach(() => {
+      container = document.createElement('div');
+      document.body.appendChild(container);
+      document.getSelection = () => {
+          return {
+            removeAllRanges: () => {},
+            getRangeAt: function() {}, 
+          };
+      };
+      jest.mock('lodash.debounce', (fn) => fn);
+    });
+
+    afterEach(() => {
+        document.body.removeChild(container);
+        container = null;
+    });
+
+    it('should call text-change event only once when editing', async () => {
+        const mockFn = jest.fn();
+        debounce.mockImplementation(fn => fn);
```

Comment:
it seems to be the same implementation as in the beforeEach

Reply:
the one in beforeEach is not working, forget to remove it. will remove it